packages/x-components/tests/e2e/common/common-steps.spec.ts

```diff
@@ -67,6 +67,10 @@ When('start button is clicked', () => {
   cy.getByDataTest('open-modal').click();
 });
 
+When('close button is clicked', () => {
```

Comment:
Clarify the button because it's in the common steps and is not any button

```suggestion
When('close modal button is clicked', () => {
```